gems/pending/VixDiskLib/VixDiskLibServer.rb

```diff
@@ -1,3 +1,9 @@
+#
+# This script doesn't run in the context of the Rails environment,
+# so the following load path manipulation is required.
+#
+$LOAD_PATH << File.expand_path(File.join(__dir__, ".."))
```

Comment:
It's run by a worker, but not as part of the worker. It runs as a simple ruby process so it won't have to load the binary libraries that the Rails app depends on - eliminating the conflict with the libs the VDDK depends on.